.github/CODEOWNERS

```
@@ -8,4 +8,4 @@
 *               @googleapis/yoshi-python
 
 
-/samples/   @googleapis/python-samples-owners
\ No newline at end of file
+/samples/   m-strzelczyk @googleapis/python-samples-owners
```

```suggestion
/samples/   @m-strzelczyk @googleapis/python-samples-owners
```

https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-code-owners#codeowners-syntax